src/store/actions/test.js

```diff
@@ -42,7 +42,7 @@ describe( 'Store', () => {
 
 		it( 'should create an action to create a block', () => {
 			const newBlock = createBlock( 'core/code', { content: 'new test text for a core/code block' } );
-			const action = actions.createBlockAction( '1', newBlock );
+			const action = actions.createBlockAction( '1', newBlock, '0' );
```

Comment:
Let's also test the `clientIdAbove` while at it, wdyt?

Reply:
addressed in 8814dc0